docs/reference/metrics.rst

```diff
@@ -0,0 +1,53 @@
+.. _reference-metrics:
+
+Metrics
+=======
+
+This page lists all supported statistics that stores gather about snaps.
+
+All metrics are generated daily. Those with seven-day windows provide the average
+averages.
+
+.. list-table::
+    :header-rows: 1
+    :widths: 1 2
+
+    * - Metric
+      - Data
+    * - ``daily_device_change``
+      - Contains the 3 series representing the number of new, continued and lost devices with the given snap installed compared to the previous day.
+    * - ``installed_base_by_channel``
+      - Contains one series per channel representing the number of devices with the
+        given snap installed, channels with no data across the entire interval are
+        omitted.
+    * - ``installed_base_by_country``
+      - Contains one series per country representing the number of devices with the
+        given snap installed.
+    * - ``installed_base_by_operating_system``
+      - Contains one series per operating_system representing the number of devices with
```

Comment:
```suggestion
      - Contains one series per operating system representing the number of devices with
```

Reply:
It wasn't me :eyes: [98d19bc](https://github.com/canonical/snapcraft/pull/5372/commits/98d19bc604c09abf310ee8e6ea100b917829f04d)